### DISCUSSION TITLE
using build_flags ok however using User_Setup failed

### DISCUSSION THREAD

**Discussion by MHz000** [Bodmer/TFT_eSPI#2810] [Compatible displays and setup files]
only if I use the following build flaggs in **platformio.ini** the display comes to live:
build_flags =
-D USER_SETUP_LOADED
-D ILI9488_DRIVER
-D TFT_MISO=19
-D TFT_MOSI=23
-D TFT_SCLK=18
-D TFT_CS=15
-D TFT_DC=2
-D TFT_RST=-1
-D LOAD_GLCD=1
-D LOAD_FONT2
-D LOAD_FONT4
-D LOAD_FONT6
-D LOAD_FONT7
-D LOAD_FONT8
-D LOAD_GFXFF
-D SMOOTH_FONT
-D SPI_FREQUENCY=27000000
However, if I make the same settings in User_Setup.h, the display remains dark. What am I doing wrong, what have I overlooked?
#define ILI9341_DRIVER
#define TFT_MISO 19 // ye
#define TFT_MOSI 23 // or
#define TFT_SCLK 18 // gn
#define TFT_CS 15 // vi Chip select control pin
#define TFT_DC 2 // bl Data Command control pin
//#define TFT_RST 4 // Reset pin (could connect to RST pin)
#define TFT_RST -1 // br => EN
#define TOUCH_CS 21
#define LOAD_GLCD // Font 1. Original Adafruit 8 pixel font needs ~1820 bytes in FLASH
#define LOAD_FONT2 // Font 2. Small 16 pixel high font, needs ~3534 bytes in FLASH, 96 characters
#define LOAD_FONT4 // Font 4. Medium 26 pixel high font, needs ~5848 bytes in FLASH, 96 characters
#define LOAD_FONT6 // Font 6. Large 48 pixel font, needs ~2666 bytes in FLASH, only characters 1234567890:-.apm
#define LOAD_FONT7 // Font 7. 7 segment 48 pixel font, needs ~2438 bytes in FLASH, only characters 1234567890:-.
#define LOAD_FONT8 // Font 8. Large 75 pixel font needs ~3256 bytes in FLASH, only characters 1234567890:-.
//#define LOAD_FONT8N // Font 8. Alternative to Font 8 above, slightly narrower, so 3 digits fit a 160 pixel TFT
#define LOAD_GFXFF // FreeFonts. Include access to the 48 Adafruit_GFX free fonts FF1 to FF48 and custom fonts
#define SMOOTH_FONT
#define SPI_FREQUENCY 27000000
#define SPI_READ_FREQUENCY 20000000
#define SPI_TOUCH_FREQUENCY 2500000

Thank you
[User_Setup-h.txt](https://github.com/Bodmer/TFT_eSPI/files/12421304/User_Setup-h.txt)

**Comment by Bodmer** [comment]
#define ILI9341_DRIVER

is not the same as:

-D ILI9488_DRIVER
